ui/.npmrc

```diff
@@ -0,0 +1,2 @@
+@fortawesome:registry=https://npm.fontawesome.com/
+//npm.fontawesome.com/:_authToken=5D54A493-EC53-4B6E-972E-5A200343CFC7
```

Comment:
Is that auth token secret? It looks like a registry key, which we should probably keep secret?

Reply:
Good catch. Switched to adding the pro icons individually as svg files